systemtests/pom.xml

```diff
@@ -0,0 +1,236 @@
+                <version>${surefire-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>system-tests</id>
+                        <goals>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <skipTests>${skipSTs}</skipTests>
+                    <includes>
+                        <include>**/*ST.java</include>
+                    </includes>
+                </configuration>
+            </plugin>
+            <!-- Used for code coverage with UT -->
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <executions>
+                    <!-- Attach JaCoCo agent to unit tests -->
+                    <execution>
+                        <id>default-prepare-agent</id>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                        <configuration>
+                            <destFile>${project.build.directory}/jacoco-unit.exec</destFile>
+                            <append>true</append>
+                        </configuration>
+                    </execution>
+                    <!-- Generate aggregated coverage report -->
+                    <execution>
+                        <id>report-aggregate</id>
+                        <goals>
+                            <goal>report-aggregate</goal>
+                        </goals>
+                        <phase>verify</phase>
+                        <configuration>
+                            <excludes>
+                                <!-- Exclude Builder classes-->
+                                <exclude>**/*Builder.class</exclude>
+                                <exclude>**/*Fluent.class</exclude>
+                                <exclude>**/*Nested.class</exclude>
+                                <!-- Exclude Constants and Enums -->
+                                <exclude>com/github/streamshub/systemtests/constants/*</exclude>
+                                <exclude>com/github/streamshub/systemtests/enums/*</exclude>
+                                <!-- Exclude Environment. It's tested by TF -->
+                                <exclude>com/github/streamshub/systemtests/Environment.class</exclude>
+                                <!-- Exclude LogCollector and Logger from code coverage -->
+                                <exclude>com/github/streamshub/systemtests/logs/*</exclude>
+                                <!-- Exclude unit test utils -->
+                                <exclude>com/github/streamshub/systemtests/unit/*</exclude>
+                                <!-- Exclude exceptions -->
+                                <exclude>com/github/streamshub/systemtests/exceptions/*</exclude>
```

Comment:
Move these several `exclude` options into `coverage/pom.xml` with the others. The rest of the `report-aggregate` execution can simply be deleted, all reporting is done by the `coverage` module. Keep the `prepare-agent` execution here though.

Reply:
Awesome, will do, thank you.